socorro/tests/processor/test_processor_app.py

```diff
@@ -99,15 +101,20 @@ def test_transform_success(self):
         pa.processor.process_crash = mocked_process_crash
         pa.destination.save_processed_crash = mock.Mock()
         finished_func = mock.Mock()
-        patch_path = "socorro.processor.processor_app.os.unlink"
-        with mock.patch(patch_path) as mocked_unlink:
-            # the call being tested
-            pa.transform("17", finished_func)
+
+        # the call being tested
+        pa.transform("17", finished_func)
+
         # test results
-        mocked_unlink.assert_called_with("fake_dump_TEMPORARY.dump")
         pa.source.get_raw_crash.assert_called_with("17")
         pa.processor.process_crash.assert_called_with(
-            "default", fake_raw_crash, fake_dumps, fake_processed_crash
+            ruleset_name="default",
+            raw_crash=fake_raw_crash,
+            dumps=fake_dumps,
+            processed_crash=fake_processed_crash,
+            # FIXME(willkg): testing this is tricky--we'd have to mock
+            # TemporaryDirectory
+            tmpdir=ANY,
```

Comment:
The value here will be something like `/tmp/tmpxidifS43` and changes every time so it's tricky to test unless we mock out parts of the tempfile module.